### PR TITLE
Add more log in pjsua_vid and vid_conf

### PR DESCRIPTION
**Pull request by sauwming**

### CODE REVIEW

File: pjsip/src/pjsua-lib/pjsua_vid.c [hunk 1]
```diff
@@ -994,6 +994,8 @@ static pj_status_t setup_vid_capture(pjsua_call_media *call_med)
     /* Done */
     inc_vid_win(wid);
     call_med->strm.v.cap_win_id = wid;
+    PJ_LOG(4,(THIS_FILE, "Call %d: video capture set up with dev %d, wid=%d",
+	      call_med->call->index, call_med->strm.v.cap_dev, wid));
```
Comment (nanangizz) [inline pjsip/src/pjsua-lib/pjsua_vid.c]: Perhaps better with call index & media index info, usually in the beginning of line: "Call %d media %d: ... " (check other logs)? Also all new logs in the same file below.

Comment (sauwming) [reply pjsip/src/pjsua-lib/pjsua_vid.c]: which new logs? This is the only one in this file.

File: pjsip/src/pjsua-lib/pjsua_vid.c [hunk 2]
```diff
@@ -1280,6 +1282,8 @@ void pjsua_vid_stop_stream(pjsua_call_media *call_med)
 	/* Decrement ref count of preview video window */
 	dec_vid_win(call_med->strm.v.cap_win_id);
 	call_med->strm.v.cap_win_id = PJSUA_INVALID_ID;
+	
+	PJ_LOG(4,(THIS_FILE, "Preview video window released"));
```
Comment (nanangizz) [inline pjsip/src/pjsua-lib/pjsua_vid.c]: this

File: pjsip/src/pjsua-lib/pjsua_vid.c [hunk 3]
```diff
@@ -1293,6 +1297,8 @@ void pjsua_vid_stop_stream(pjsua_call_media *call_med)
 	/* Decrement ref count of stream video window */
 	dec_vid_win(call_med->strm.v.rdr_win_id);
 	call_med->strm.v.rdr_win_id = PJSUA_INVALID_ID;
+
+	PJ_LOG(4,(THIS_FILE, "Stream video window released"));
```
Comment (nanangizz) [inline pjsip/src/pjsua-lib/pjsua_vid.c]: and this.

**Pull request merged**